### DISCUSSION TITLE
Shortcuts for Pull Requests never updates

### DISCUSSION THREAD

**Discussion by carestad** [community/community#73538] [Mobile]
I have tried again and again to create shortcuts on my home screen in the app directly to all (or selected) repositories and their pull request, but the list never updates with new PRs, for example from Dependabot. Are there any limitations here I am not aware of?

**Comment by alcere** [comment]
Hi @carestad! Could you please provide us with your GitHub app version and the platform you are using?

**Reply by carestad** [reply]
> Could you please provide us with your GitHub app version and the platform you are using?

Hi!

The app is on version 1.137.0 (755), with SDK 33. The mobile phone model is Samsung SM-S901B.

I use the app for both GitHub.com and an enterprise instance at work, but these shortcuts don't update for me on either platform unfortunately.

**Reply by alcere** [reply]
Thank you for the information!

**Reply by alcere** [reply]
@carestad I've looked into this, and I wasn't able to reproduce the issue when I created a shortcut scoped to a single repository. The steps I followed were:
- Select Shortcuts 
- Create Shortcut
- Select a specific repository as the Scope.
- Select PULL REQUEST as the type.

Then, I could see all pull requests in that repo, and any new pull request created in that repo. Did I reproduce the scenario you referred to as "selected repository" correctly?

Regarding the other scenario you mentioned, creating a shortcut scoped to all repositories, I was able to reproduce the issue and I am currently investigating the reasons for this behavior, thanks for reporting!

**Reply by carestad** [reply]
Thanks for looking into this!

I can confirm that selecting a single repository works fine here as well. The "all repositories" option is the one that indeed seems to be struggling a little bit.

**Reply by alcere** [reply]
After taking some more time to look into it, it turns out the behaviour you described is the expected one. When we set the scope as ALL REPOS, it means ALL REPOS the user has access to, including those the user is a member of and all public repos within GitHub. As you might expect, having access to all issues, pull requests or discussions within GitHub isn't very useful, so we imposed a filter that sets some kind of relation between the user and the results. This is the second filter you see starting from the left: "Created" (created by me), "Assigned" (assigned to me), etc. We understand that the labels we use for the different values in this filter can be confusing, so we have started an internal discussion within the team to improve them. We apologize for any confusion this may have caused and we will soon be releasing an update to avoid this kind of misunderstanding. Stay tuned for future changes!

**Reply by carestad** [reply]
Ah, okay. Thanks for the update!

Could it be possible to allow the user to select more than one repo instead? I would actually very much love to have a shortcut that just shows me every issue or pull request for the X selected repositories though, especially when it comes to dependabot updates etc. that tends to gather up and across multiple repositories.

**Reply by alcere** [reply]
I think that's a very interesting idea! I will open an internal issue to explore its feasibility. Thanks for this idea, and please keep the feedback coming!

**Comment by kauaneiras** [comment]
🚀

**Comment by xbz77** [comment]
- [x] @Kskskekk_[[]()]()_
